packages/sui-theme/README.md

```diff
@@ -63,3 +63,39 @@ Find below de compat varible groups available:
 ## Update
 
 If you need to update any of these variables please pull request.
+
+
+
+## Utility classes
+
+**Spacing classes**
+
+Provides a basic spacing classes library based in emmet convection.
+
+In your vertical theme import the library:
+
+```scss
+@import ~/@s-ui/theme/lib/utils/_spacing.scss
```

Comment:
IMHO. I would create a new directory called `utility-classes`

So, it ends up like: `@import ~/@s-ui/theme/lib/utility-classes/_spacing.scss`